src/extension/importer/web/src/main/java/org/geoserver/importer/web/ImportTaskTable.java

```diff
@@ -449,11 +451,21 @@ public LayerPreviewPanel(String id, IModel<ImportTask> model) {
                             .setNullValid(false)
                             .setOutputMarkupId(true));
 
-            String javascript =
-                    "go(document.getElementById('" + get("links").getMarkupId() + "'));";
-            add(
-                    new ExternalLink("go", "#")
-                            .add(new AttributeModifier("onclick", new Model<>(javascript))));
+            add(new ExternalLink("go", "#").setOutputMarkupId(true));
+        }
+
+        @Override
+        public void renderHead(IHeaderResponse response) {
```

Comment:
This one is not working. When importing multiple layers, the "go" links all make the first layer preview appear. E.g., I've imported the tasmania shapefiles in another workspace, after a successful import, all "go" link preview the cities layer:

![image](https://github.com/user-attachments/assets/a0129540-a4f1-4fa1-b49f-1c58ba0cc88e)

Reply:
I looked at the code and I'm wondering if this behavior you're describing is actually the current behavior without this PR?

Reply:
There's also still more inline script on this page that I need to remove.

Reply:
> I looked at the code and I'm wondering if this behavior you're describing is actually the current behavior without this PR?

That may well be true, I haven't used the importer GUI in a long while (and did not test it before this change).

Reply:
Tested again and found it working... duh, not sure what happened here.